src/runtime/zarf.yaml

```diff
@@ -0,0 +1,28 @@
+kind: ZarfPackageConfig
+metadata:
+  name: uds-core-runtime
+  description: "UDS Core Runtime"
+  url: "https://github.com/defenseunicorns/uds-runtime"
+
+components:
+  - name: uds-runtime
+    required: false
+    images:
+      - ghcr.io/defenseunicorns/uds-runtime:0.4.0
+    charts:
+      - name: uds-runtime
+        namespace: uds-runtime
+        version: "v0.4.0"
```

Comment:
Should this not be 0.1.0? Aligned with https://github.com/defenseunicorns/uds-runtime/blob/main/chart/Chart.yaml#L18 ?

Reply:
This is actually the git tag/version for the repo. Slightly confusing, but makes sure that we are pulling the chart for the same image. Might be good for runtime to synchronize the version of the chart as well - cc @UncleGedd ?

I do believe this works as is, even if not the most obvious - it does deploy in CI and locally as expected.

Reply:
Made an issue to address: https://github.com/defenseunicorns/uds-runtime/issues/355